src/docs/components/modals.vue

```diff
@@ -19,6 +27,21 @@
         dh2oModal('modalJs', true, (isShowing) => {
           this.isModalJsShowing = isShowing
         })
+      },
+
+      showModalLock () {
+        this.$refs.modalLock.show()
+        this.lock()
+      },
+
+      lock() {
+        this.$refs.modalLock.lock()
+        this.isModalLockLocked = true
```

Comment:
If you added a property on the element you wouldnt have to manage that yourself. We could add something like isLocked() and it would return true/false